src/main/java/net/minecraftforge/network/simple/SimpleChannel.java

```diff
@@ -223,6 +246,17 @@ public MessageBuilder<MSG> noResponse()
             return this;
         }
 
+        /**
+         * Set the message consumer, which is called once a message has been decoded. This accepts the decoded message
+         * object and the message's context.
+         *
+         * The consumer is called on the network thread, and so should not interact with most game state by default.
+         * {@link NetworkEvent.Context#enqueueWork(Runnable)} can be used to handle the message on the main server or
+         * client thread.
```

Comment:
Paragraphs other than the first should either be separated by or surrounded with <p> tags. (I personally prefer the latter.)

```suggestion
         * <p>The consumer is called on the network thread, and so should not interact with most game state by default.
         * {@link NetworkEvent.Context#enqueueWork(Runnable)} can be used to handle the message on the main server or
         * client thread.</p>
```